source/includes/administration/_users.md

```diff
@@ -123,21 +158,20 @@ Retrieve information about a specific user. If you want access to other users in
 Attributes | &nbsp;
 ---------- | -----------
 `id`<br/>*UUID* | The id of the user
-`userName`<br/>*string* | The username of the user
+`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the user was created
+`email`<br/>*string* | The email of the user
+`environments`<br/>*Array[[Environment](#administration-environments)]* | The environments the user is member of<br/>*includes*: `id`, `name`
 `firstName`<br/>*string* | The first name of the user
 `lastName`<br/>*string* | The last name of the user
-`email`<br/>*string* | The email of the user
-`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the user was created
-`status`<br/>*string* | The current status of the user.
-`primaryRoleBinding`<br/>*RoleBinding* | The primary role assigned to this user. This role will always be a fixed role.
 `organization`<br/>*[Organization](#administration-organization)* | The organization to which the user belongs
-`environments`<br/>*Array[[Environment](#administration-environments)]* | The environments the user is member of<br/>*includes*: `id`, `name`
+`primaryRoleBinding`<br/>*RoleBinding* | The primary role assigned to this user. This role will always be a fixed role.
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles that are assigned to the user<br/>*includes*: `id`, `name` and `environment.id`
-
+`serviceConnection`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The service connection of the user<br/>*includes*: `id`, `name`, `serviceCode`, `type`, and `state`
+`status`<br/>*string* | The current status of the user.
+`userName`<br/>*string* | The username of the user
```

Comment:
Is there any particular ordering of the fields? Should we put things like the this on top and Json objects, arrays after?

Reply:
I sorted them a-z. IMO it's easier to find variables like that. Is there something the team uses by default?

Reply:
I am not exactly sure if there is a default. I usually arrange it in the order they appear in the response object. But sorted order makes sense too.